.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,33 @@
+name: Dependabot auto-merge
```

Comment:
Could you name this or clearly comment it (for now) to make it clear that this is for metadata exploration rather than actually doing the auto-merge?